zilliqa/src/api/eth.rs

```diff
@@ -1,6 +1,9 @@
 //! The Ethereum API, as documented at <https://ethereum.org/en/developers/docs/apis/json-rpc>.
 
-use std::sync::{Arc, Mutex, MutexGuard};
+use std::{
+    sync::{Arc, Mutex, MutexGuard},
+    time::Instant,
```

Comment:
Can we use `zilliqa::time::SystemTime` instead since we can control it in tests? (and this would let you write a test for filter expiration)